frontend/app/package.json

```diff
@@ -44,7 +44,6 @@
     "@luma.gl/shadertools": "^8.5.18",
     "@luma.gl/webgl": "^8.5.18",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
-    "@streamlit/lib": "1.0.0",
```

Comment:
@streamlit/lib is no longer being consumed (because it's relying on the contents of lib directly at the same git commit)